src/user/proposals/UserProposalDashboard/DashboardContent/DashboardContent.scss

```diff
@@ -177,3 +177,36 @@
     }
   }
 }
+
+@media (min-width: 320px) and (max-width: 1024px){
+  .dashboard-content {
+    padding: 0;
+    height: 100%;
+    .searchbar-container {
+      .button-container {
+        .posts {
+          width: 100%;
+          .category {
+            text-align: center;
+            button {
+              margin: 5px;
+            }
+          }
+        }
+      }
+    }
+    .proposal-container{
+      .proposals {
+        .single-proposal {
+          .proposal-content {
+            .proposal-options {
+              .option-btn {
```

Comment:
Not a good practice. Although this might require changing how the HTML is written

Reply:
Which one are you saying about? May I know?

Reply:
@aayush-05, I'm not sure what you said. Please elaborate!

Reply:
Was talking about nesting; max 3 levels is what I suggest

Reply:
I did that on purpose because the class variable names are being repeated multiple times(in different files) so I had to nest to override.